compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc

```diff
@@ -0,0 +1,72 @@
+== Scan Images in GitLab Container Registry
+
+Configure Prisma Cloud to scan your GitLab Container Registry on GitLab EE without using administrator credentials.
```

Comment:
To be confirmed by Engineering, if we support scanning on just GitLab EE or on CE as well!

Reply:
GCR scan is supported on both EE and CE.

Reply:
so we need to change the sentence here, right?

Reply:
Thanks Tal, I updated it with just GitLab to indicate both EE and CE.